### DISCUSSION TITLE
Feature Suggestion: Add the Bestiary tab to use-cases for Search Strings.

### DISCUSSION THREAD

**Discussion by mal3diction** [Lailloken/Exile-UI#181] [Ideas & Feedback]
In the Challenges menu (def. hotkey: H) there's a tab for Bestiary that shows all of your captured beasts. It supports regex and I use a few to sort between expensive, moderate, and cheap-but-movable beasts.  

It would be fantastic if this window was a supported use-case for the Search Strings because I love the context sensitive middle/right click instead of having to find increasingly more crazy hotkey combos for Awakened.

**Comment by Lailloken** [comment]
Sounds good, I'll implement it for the next release.

**Comment by Lailloken** [comment]
I have just released v1.29.6 which includes a new 'bestiarydex' use-case for search-strings.
